README.md

```diff
@@ -174,12 +174,13 @@ Options:
                                                  Preview TFM. See
                                                  https://dotnet.microsoft.com/platform/support/policy/dotnet-core for
                                                  details for what these mean.
+                                                                    --format                                        Specify what format desired. Currently supports html and sarif. 
```

Comment:
The formatting of this line is super odd, can you align it with other lines above? Also use the preview tab above to see how the text will look like so you can get a better idea how it looks visually.